movement/movement_config.h

```diff
@@ -29,6 +29,7 @@
 
 const watch_face_t watch_faces[] = {
     simple_clock_face,
+    totp_face_lfs,
```

Comment:
I assume this is just for testing but this face should be removed from here before it is merged in to not change the defaults for anyone.